action.yml

```diff
@@ -27,32 +27,21 @@ inputs:
   vertex_machine_type:
     description: 'Type of Vertex AI machine to run notebooks on e.g. n1-standard-4'
     default: 'n1-standard-4'
+    required: false
   region:
     description: 'Google Cloud region e.g. us-central1, us-east4'
     default: 'us-central1'
-
-permissions:
-  pull-requests: 'write'
-  issues: 'write'
-  contents: write
-  id-token: 'write'
-
-env:
-  VERTEX_CONTAINER: 'gcr.io/deeplearning-platform-release/base-cu110:latest'
```

Comment:
Do users need to know these values now, or is there a sane default?

Reply:
They are set to a sane default. Defining them in an 'env' section didn't seem to work - either there is some quirk with this in composite actions or I'm doing something wrong. I could move them to the inputs section but they would always use the default and may lead to confusion.

Reply:
The notebook executor is in that container. The only reason we'd change it is if Vertex switched to a new container. Same for the URLs, those are Vertex's defined URLs, not something we should expect a user to update.